README.md

```diff
@@ -1,8 +1,12 @@
 # Elastic Cloud Control
 
-`ecctl` is the CLI on steroids for the Elastic Cloud API. It wraps common operations commonly needed by Elastic Cloud operators within a single command line tool.
+NOTE: **`ecctl` is still in beta and should be considered experimental!**
```

Comment:
There is a warning message we normally use. See for example ECK [docs](https://www.elastic.co/guide/en/cloud-on-k8s/current/index.html#k8s-deploy-eck) where there is a banner saying "You are looking at the documentation for a beta release." and in case of ecctl we should also specifically highlight that things might change since we know we want to migrate to use the `/deployments` endpoint before it will be GAed.